azavea/ism-watershed-wellness-snapshot
# Use default sensor value for missing variables

## Overview

If a sensor has live API data available, but is not reporting all of the variables needed for the app, use the default values for the missing variable.

Connects #106 

### Demo

`develop`:
![image](https://user-images.githubusercontent.com/1042475/62724641-71d3d900-b9e1-11e9-8241-43ea905208b5.png)

This branch:
![image](https://user-images.githubusercontent.com/1042475/62724625-68e30780-b9e1-11e9-98e1-938ad1ea2425.png)

## Testing Instructions

- Visit the sensor overview pane for the Tinicum sensor on the deploy preview for this branch and `develop`.
- Verify that the turbidity value is set on this branch with the default sensor value.
